_tests/apps_test.go

```diff
@@ -1,26 +1,32 @@
 package _tests_test
 
 import (
+	"fmt"
 	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/config"
 	. "github.com/onsi/gomega"
 )
 
+func getRandAppName() string {
+	return fmt.Sprintf("apps-test-%d", GinkgoConfig.RandomSeed)
```

Comment:
Yes, good call that I totally glossed over. We can keep it simple and stick with a random int here. Changes incoming.